web_src/js/index.js

```diff
@@ -3337,3 +3337,135 @@ window.onOAuthLoginClick = function () {
     oauthNav.show();
   }, 5000);
 };
+(function () {
```

Comment:
I prefer to create a new js file and put all the new functions there but compiled into /public/js/index.js